dashboard/config/foorm/forms/surveys/teachers/teacher_end_of_year_survey.0.json

```diff
@@ -2,34 +2,28 @@
  "title": "Teacher end-of-year survey CSF, CSD, CSP",
```

Comment:
nit: is this final title?  Should it be "**for** CSF, CSD, CSP" maybe?

Reply:
Is this title actually exposed to the user?  If so it should be "Code.org End-of-Year Teacher Survey 2020"

Reply:
The title is exposed to the user. You can either not have a title (then nothing will show up) or update it

Reply:
updated to suggested title